src/IssueCreator/Dialogs/IssueTemplates.cs

```diff
@@ -0,0 +1,44 @@
+﻿using IssueCreator.Logging;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace IssueCreator.Dialogs
+{
+    public partial class IssueTemplates : IssueCreator.Dialogs.ManageIssue
+    {
+        public IssueTemplates() : base()
+        {
+            InitializeComponent();
+        }
+
+        public IssueTemplates(IssueManager issueManager, Settings settings, FileLogger logger)
+            : base(issueManager, settings, logger, null)
+        {
+            InitializeComponent();
+        }
+
+        private void IssueTemplates_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        private async void loadIssue_Click(object sender, EventArgs e)
+        {
+            if (!this.ValidateChildren())
+            {
+                return;
+            }
+
+            await LoadIssueDetails();

```

Comment:
The returns before the Issue is loaded in a cache miss

Reply:
Does this happen during a cache miss when the issue does not exist, or regardless?

Reply:
When the issue exists.

Reply:
Interesting. I'll try to repro this... If you happen to catch the exception in the debugger that would help 🙂.

Reply:
repro steps
- ensure there is no cache entry for a given issue
- enter the issue number in the template dialog
- hit Enter (rather than navigating away from the input textbox to minimize the time between issue load and dialog close)

Reply:
Figured it out -- missing `.ConfigureAwait(false)` on the calls that are not happening on the UI thread.